lib/vanagon/platform/windows.rb

```diff
@@ -177,7 +177,7 @@ def generate_msi_package(project)
         app_heat_flags = " -dr INSTALLDIR -v -ke -indent 2 -cg AppComponentGroup -gg -srd -t wix/filter.xslt -sreg -var var.AppSourcePath "
         app_source_path = "SourceDir/#{project.settings[:base_dir]}/#{project.settings[:company_id]}/#{project.settings[:product_id]}"
         appdata_heat_flags = "  -dr APPDATADIR -v -ke -indent 2 -cg AppDataComponentGroup -gg -srd -t wix/filter.xslt -sreg -var var.AppDataSourcePath "
-        appdata_source_path = "SourceDir/CommonAppDataFolder"
+        appdata_source_path = "SourceDir/CommonAppDataFolder/#{project.settings[:company_id]}"
```

Comment:
Shouldn't this also require a change in https://github.com/puppetlabs/vanagon/blob/master/resources/windows/wix/directorylist.wxs.erb#L12L14 ?